### PR TITLE
Docs: update information about point 8 Tests

### PR DESCRIPTION
**Pull request by medleycodacy**

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -68,7 +68,17 @@ Please make sure you have these tools installed before starting this process:
 
 -   [ ] 8.  Test
 
-    Test the entire release.
+    -   Validate that your feature is working according to the requirements
```
Comment (paulopontesm) [inline RELEASE.md]: Validate that the features present in changelog generated during the circleci pipeline work according to the requirements.

File: RELEASE.md [hunk 2]
```diff
@@ -68,7 +68,17 @@ Please make sure you have these tools installed before starting this process:
...
+
+    -   Do exploratory testing around the functionalitys your feature impacted
```
Comment (paulopontesm) [inline RELEASE.md]: functionalitys -> functionalities

Comment (paulopontesm) [reply RELEASE.md]: Do some exploratory tests to find previously existing features that might have impacted by these changes

Comment (paulopontesm) [inline RELEASE.md]:
```suggestion
    -   Do some [exploratory tests](https://handbook.dev.codacy.org/product/engineering/QA/levels.html#exploratory-testing) to find previously existing features that might have impacted by these changes
```

File: RELEASE.md [hunk 3]
```diff
@@ -68,7 +68,17 @@ Please make sure you have these tools installed before starting this process:
...
+        -   More information about exploratory testing [Here](https://handbook.dev.codacy.org/product/engineering/QA/levels.html).
```
Comment (paulopontesm) [inline RELEASE.md]:
```suggestion

```

File: RELEASE.md [hunk 4]
```diff
@@ -68,7 +68,17 @@ Please make sure you have these tools installed before starting this process:
...
+
+    -   Add the new tests to the test suite
```
Comment (paulopontesm) [inline RELEASE.md]:
```suggestion
    -   If you find any critical path that might have been affected, make sure you add/edit the tests in our [automation test suite](https://bitbucket.org/qamine/qa-automation-tests/)
```

File: RELEASE.md [hunk 5]
```diff
@@ -68,7 +68,17 @@ Please make sure you have these tools installed before starting this process:
...
+
+    -   Run the Regression tests 
+        -   More information about how to run [Here](https://bitbucket.org/qamine/qa-automation-tests/src/master/).
```
Comment (paulopontesm) [inline RELEASE.md]:
```suggestion
    -   Run the regression tests with our [automation test suite](https://bitbucket.org/qamine/qa-automation-tests/src/8aa6640db54f8cf3ac4f07b70647d66a0ec49739/docs/getting-started.md#run-the-tests). 
```

File: RELEASE.md (hunk 1 above)
Comment (medleycodacy) [reply RELEASE.md]: done

File: RELEASE.md (hunk 2 above)
Comment (medleycodacy) [reply RELEASE.md]: done

File: RELEASE.md (hunk 3 above)
Comment (medleycodacy) [reply RELEASE.md]: done

File: RELEASE.md (hunk 4 above)
Comment (medleycodacy) [reply RELEASE.md]: done

File: RELEASE.md (hunk 5 above)
Comment (medleycodacy) [reply RELEASE.md]: done

**Pull request merged**